src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelDelegateCommands.cs

```diff
@@ -122,6 +122,9 @@ private void InitializeDelegateCommands()
         public DelegateCommand SaveImageCommand { get; set; }
         public DelegateCommand ShowSaveImageDialogAndSaveResultCommand { get; set; }
         public DelegateCommand ToggleConsoleShowingCommand { get; set; }
+        // TODO: To be removed in Dynamo 2.0
+        [System.Obsolete("This delegate command is obsolete, set DynamoViewModel.ShowPreviewBubbles directly instead.")]
+        public DelegateCommand TogglePreviewBubblesShowingCommand { get; set; }
```

Comment:
I just tested this and it turns out I still need to hook it up or make referencing it not compile-able. It could not be a dummy command at least not in this way. I will make a commit just to hook it to the old method.